### PR TITLE
Initial implementation

### PR DESCRIPTION
**Pull request by poteto**

### CODE REVIEW

**Comment by poteto** [comment]
cc @danmcclain

**Comment by christopheradams** [comment]
Cool

File: README.md [hunk 1]
```diff
@@ -22,3 +22,38 @@ If [available in Hex](https://hex.pm/docs/publish), the package can be installed
...
+defmodule MyApp.Terraformers.Foo do
+  # example client made with HTTPoison
+  alias MyApp.Clients.Foo
+  import Plug.Conn
+  import Terraform, only: [send_response: 1]
+  
+  def init(opts), do: opts
+
+  # match specific path
+  def call(%{method: "GET", request_path: "/v1/hello-world"} = conn, _) do
```
Comment (poteto) [inline README.md]: update example

File: README.md [hunk 2]
```diff
@@ -22,3 +22,38 @@ If [available in Hex](https://hex.pm/docs/publish), the package can be installed
...
+    send_resp(conn, 200, "Hello world")
+  end
+  # match all `get`s
+  def call(%{method: "GET", params: params, req_headers: req_headers} = conn, _) do
```
Comment (poteto) [inline README.md]: update example

**Pull request merged**